tools/test.py

```diff
@@ -325,12 +329,10 @@ def HasRun(self, output):
     # duration_ms is measured in seconds and is read as such by TAP parsers.
     # It should read as "duration including ms" rather than "duration in ms"
     logger.info('  ---')
-    logger.info('  duration_ms: %d.%d' %
+    logger.info('  duration (s): %d.%d' %
```

Comment:
I know this is an annoyance for many folks but I'd advise against changing this and instead learn to read it as "duration including milliseconds". From memory there's some tooling in Jenkins that depends on this. It's become a TAP quirk, not something we've introduced.

Reply:
Ack. I'm just smoke testing the CI TAP tooling

Reply:
Weird. I didn't make this change... Oh, @refack pushed to my branch. Hmmmm.....

Yeah, `duration_ms` is a TAP thing and you will break stuff if you change it. Please don't!  :-D

Reply:
(I think I like all the other changes @refack made.)